packages/@sanity/base/src/search/weighted/applyWeights.ts

```diff
@@ -3,55 +3,98 @@ import {SearchHit, WeightedHit, SearchSpec} from './types'
 
 type SearchScore = [number, string]
 
-// takes a set of terms and a value and returns a [score, story] pair where score is a value between 0, 1 and story is the explanation
+/**
+ * Calculates a score (between 0 and 1) indicating general search relevance of an array of
+ * search tokens within a specific string.
+ *
+ * @internal
```

Comment:
The smallest of nits: the `@internal` (or public or whichever one of the "availability tags") tag should be at the bottom of the comment (the last thing) :)